Digital-launcher.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec "/app/lib/python" "-OO" "/app/Digital.py" "$@"
```

Comment:
The game originally came with a script for a launcher, the command it effectvely runs is this:

```
exec "`dirname \"$0\"`/lib/python" "-OO" "${0%.sh}.py" "$@"
```

That file became a problem to patch because it assumes everything is on the same folder as itself, which is how the game is distributed, and I had moved the launcher to /app/bin.
While I had some luck using sed to patch the part where it calls the python command, I failed patching the part where it calls the .py file. Now that you suggested moving the .py to /app/bin too, I will try again the sed option.

On needing a launcher or not: It looks like the launcher enables optimizations and allows passing arguments to python